### ISSUE TITLE
[HL 25] Machine Gun has 0 primary ammo, but has secondary shell ammo, the machine gun becomes inaccessible

### ISSUE THREAD

**Issue by ChrisCostes** [ValveSoftware/halflife#3611]
Hello,

I am writing to report a bug that I encountered while playing the 25th Anniversary Half-Life Multiplayer. I am creating a seperate request for each issue per "kisak-valve"'s request:

When the machine gun has 0 primary ammo, but has secondary shell ammo, the machine gun becomes inaccessible. This seems to be a result of a fix for a zero ammo issue that was introduced in the 25th anniversary update, but it also affects the secondary ammo function of the machine gun. The machine gun should be usable when it has shell ammo, regardless of the primary ammo count.

Please inform the programmers of these bugs and fix them as soon as possible. Thank you for your attention and your great work on the 25th Anniversary Half-Life Multiplayer.

Thank you for your attention to this isssue,
-Chris C.

**Comment by SamVanheer** [comment]
Probably a duplicate of #3001. This was already happening before the update.

**Comment by ChrisCostes** [comment]
To whom it may concern:

I can confirm that this issue was fixed with the November 29, 2023 Half-Life update. I tested the Machine Gun by firing the gun until 0 bullets, but had 2 shells and the Machine Gun was still accessible to fire the remaining shell ammo.

I would like to express my sincere thanks to Valve employees for a timely resolution to this issue!

P.S. Thanks Gabe for the 25th Anniversary Half-Life update, here’s to the next 25 years!

Sincerely,

Chris Costes 😊
